pom.xml

```
@@ -212,11 +213,11 @@
         <maven-deploy-plugin.version>3.0.0-M1</maven-deploy-plugin.version>
         <maven-install-plugin.version>3.0.0-M1</maven-install-plugin.version>
         <maven-javadoc-plugin.version>3.0.1</maven-javadoc-plugin.version>
-        <jacoco-maven-plugin.version>0.8.2</jacoco-maven-plugin.version>
         <maven-site-plugin.version>3.7.1</maven-site-plugin.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
         <maven-surefire-plugin.version>3.0.0-M2</maven-surefire-plugin.version>
         <mockito.version>2.23.4</mockito.version>
+        <powermock.version>2.0.0</powermock.version>
```

```suggestion
        <objenesis.version>3.0.1</objenesis.version>
        <powermock.version>2.0.0</powermock.version>
```

in addition of adding Objenesis to dependency management and removing the exclusion.